vignettes/basic-usage.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Basic Usage of mrgvalidate"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{basic_usage}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r options, include = FALSE}
+options(rmarkdown.html_vignette.check_title = FALSE)
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  echo = TRUE,
+  message = FALSE
+)
+library(knitr)
+library(kableExtra)
+if (interactive()) {
+  devtools::load_all()
+}
+```
+
+# Introduction
+
+The purpose of the `mrgvalidate` package is to generate 7 specific documents that are necessary for the software validation process at Metrum Research Group. Those documents are:
+
+* release-notes.docx
+* validation-plan.docx
+* validation-testing.docx
+* requirements-specification.docx
+* traceability-matrix.docx
+* testing-results.docx
+* validation-summary.docx
+
+This vignette demonstrates how to generate these with a few simple library calls.
+
+```{r load}
+library(mrgvalidate)
+```
+
+# Generating the docs
+
+By default, these calls (`create_package_docs()` and `create_metworx_docs()`) will write `.docx` files for all six documents into your working directory. Parameterized `.Rmd` files are first copied over, but are removed at the end (you can override this with the argument `cleanup_rmd = FALSE`). If all input data conforms to what is described in [`?mrgvalidate::input_formats`](https://metrumresearchgroup.github.io/mrgvalidate/reference/input_formats.html), you should only have to write a single a line.
+
+## Usage: Packages
+
+```{r create_package_docs, eval = FALSE}
+create_package_docs(
+  product_name = "Fake Product", 
+  version = "vFake", 
+  specs = spec_df,                # a tibble containing stories and requirements to validate
```

Comment:
missing `repo_url` arg in this example. (Also missing `output_dir`, though it's not required)